### DISCUSSION TITLE
tutorials of PNP Site Core Library

### DISCUSSION THREAD

**Discussion by quantrpeter** [pnp/pnpframework#183] [General]
Hi all
  Where i can find tutorials of PNP Site Core Library?
Thanks
Peter

**Comment by jansenbe** [comment]
PnP Sites Core is retired and replaced by PnP Framework, so please use that going forward. Both these libraries don't have extensive documentation due to historical reasons. We're also building a brand new library ([PnP Core SDK](https://github.com/pnp/pnpcore/tree/dev) for which we'r doing a better job when it comes to documentation. Best way to learn is from checking the code (eg PnP PowerShell uses a lot of the PnP Framework pieces) and ask questions. 

The harder part might be getting started with auth, for that we do have some docs: https://github.com/pnp/pnpframework/blob/dev/docs/MigratingFromPnPSitesCore.md
